python_modules/dagster/dagster_tests/core_tests/structured_config_tests/test_structured_config_types.py

```diff
@@ -386,7 +395,7 @@ def a_job():
         )
 
 
-def test_nested_discriminated_unions():
+def test_nested_discriminated_unions() -> None:
```

Comment:
not super important but given that every test is going to be annotated with `-> None` I'm not sure if it adds much / is necessary